src/harness/testcases/WINNF_FT_S_GRA_testcase.py

```diff
@@ -314,21 +315,20 @@ def generate_GRA_5_default_config(self, filename):
         'registrationData': [conditional_parameters]
     }
 
-    cbsd_record = [device_c1]
-    grant_record = [grant_g1]
+    cbsd_records = [device_c1]
+    grant_records = [grant_g1]
 
     sas_harness_config = {
         'sasTestHarnessName': 'SAS-Test-Harness-1',
         'hostName': 'localhost',
         'port': 9001,
-        'serverCert': "certs/server.cert",
-        'serverKey': "certs/server.key",
-        'caCert': "certs/ca.cert"
+        'serverCert': os.path.join('certs', 'server.cert'),
+        'serverKey': os.path.join('certs', 'server.key'),
+        'caCert': os.path.join('certs', 'ca.cert')
     }
     sas_harness_dump_records = {
-        'cbsdRecords': generateCbsdRecords(cbsd_record, [grant_record])
+        'cbsdRecords': generateCbsdRecords(cbsd_records, [grant_records])
```

Comment:
Hm, I'm now seeing that my earlier suggestion was not the best: it's now not very clear why `grant_records` is being wrapped in a list while `cbsd_records` is not. One way to fix it would be to do the following:

```
  'cbsdRecords': generateCbsdRecords([device_c1], [ [grant_g1] ])
```

since you don't actually use `cbsd_records` and `grant_records` anywhere else. Then it's a bit clearer what's going on. Sorry for the back-and-forth!

Reply:
Updated   accordingly